src/code_review_agent/integrations/github_client.py

```
@@ -0,0 +1,397 @@
+            repo = self._github.get_repo(repo_full_name)
+            pr = repo.get_pull(pr_number)
+            
+            logger.debug(
+                "Retrieved pull request",
+                repo=repo_full_name,
+                pr_number=pr_number,
+                title=pr.title,
+                state=pr.state
+            )
+            
+            return pr
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)